src/commands/analyze-revenue.cmd.ts

```diff
@@ -0,0 +1,178 @@
+import { CliUtilityService, Command, CommandRunner, Option } from 'nest-commander'
+import { Logger } from '@nestjs/common'
+import { AuctionResult, AuctionValidator, DsSamSDK, InputsSource, SourceDataOverrides } from '@marinade.finance/ds-sam-sdk'
+import fs from 'fs'
+
+const COMMAND_NAME = 'analyze-revenues'
+
+type AnalyzeRevenuesCommandOptions = {
+  inputsCacheDirPath: string
+  resultsFixtureFilePath: string
+  snapshotValidatorsFilePath: string
+  resultsFilePath?: string
+}
+
+export type SnapshotValidatorMeta = {
+  vote_account: string
+  commission: number
+  mev_commission?: number
+  stake: number
+  credits: number
+}
+
+export type SnapshotValidatorsCollection = {
+    epoch: number
+    slot: number
+    capitalization: number
+    epoch_duration_in_years: number
+    validator_rate: number
+    validator_rewards: number
+    validator_metas: SnapshotValidatorMeta[]
+}
+
+export type RevenueExpectation = {
```

Comment:
It could be good to store `epoch` and `slot` within the output JSON file to add a sanity check into the settlement processing engine that a JSON created from a different epoch is not loaded.

Reply:
@ochaloup please note that output interface is not changed